hknweb/reviewsessions/templates/reviewsessions/index.html

```diff
@@ -0,0 +1,97 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block title %}Review Sessions{% endblock %}
+
+{% block header %}
+<!-- Scripts which make the calendar work:-->
+<link rel="stylesheet" href="{% static "reviewsessions/style.css" %}">
+<link rel="stylesheet" href="https://unpkg.com/@fullcalendar/core/main.css" />
+<link rel="stylesheet" href="https://unpkg.com/@fullcalendar/daygrid/main.css" />
+<link rel="stylesheet" href="https://unpkg.com/@fullcalendar/timegrid/main.css" />
+<link rel="stylesheet" href="{% static "css/tachyons.css" %}" />
+
+<script src="https://unpkg.com/@fullcalendar/core/main.js"></script>
+<script src="https://unpkg.com/@fullcalendar/daygrid/main.js"></script>
+<script src="https://unpkg.com/@fullcalendar/timegrid/main.js"></script>
+
+<script type="text/javascript">
+    document.addEventListener('DOMContentLoaded', function() {
```

Comment:
This code is incredibly similar to [events calendar code](https://github.com/compserv/hknweb/blob/46fecc97931d7d0af714a042bc4eae0035204ff1/hknweb/events/templates/events/index.html#L19). Let's merge them into one template and reference that template in both events and in review sessions.

Reply:
[Jira ticket](https://hkn-mu.atlassian.net/secure/RapidBoard.jspa?rapidView=1&projectKey=COMPSERV&modal=detail&selectedIssue=COMPSERV-40) created.